CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+| `src`             | Contains all the files needed to build the site using 11ty. This includes the content for all pages and news articles, CSS files, layout and partial templates, and JavaScript files.                                               |
+| `.eleventy.js`    | The Configuration file used by 11ty in order to build the site. Specifies the collections for 11ty, [passthrough copy files](https://www.11ty.dev/docs/copy/), and other settings including the source folder for input and output. |
+| `.eleventyignore` | A list of files and folders which are ignored by Eleventy while building the site                                                                                                                                                              |
+| `.gitignore`      | A list of files and folders which won't be tracked by Git.                                                                                                                                                                          |
+| `.eslintrc.json`  | [ESLint configuration data](https://eslint.org/docs/user-guide/configuring).                                                                                                                                                        |
+| `.eslintignore`   | A list of files and folders which won't be passed through ESLint                                                                                                                                                                    |
+| `AUTHORS.md`      | List of copyright holders and contributors                                                                                                                                                                                          |
+| `Gruntfile.js`    | Grunt file used to copy data from node_modules and do linting process.                                                                                                                                                              |
+| `package.json`    | Contains scripts and a list of dependencies required to build the site, as well as general information about the repository                                                                                                                                                                             |
+| `README.md`       | Read Me file provides introduction to the repository.                                                                                                                                                                               |
+
+**Contributions to the project can be made in following ways:**
+
+1. *Contribution towards building features for the project.*
+
+* There are several features that still require attention.
+  * Improving the User interface of submitted answers.
+  * Search functionality among the site data.
+  * Support for Comments.
+
```

Comment:
This is not required and can be deleted.

Reply:
fixed